### DISCUSSION TITLE
Avoid fzf closing after selection

### DISCUSSION THREAD

**Discussion by Alberto-Rivera** [junegunn/fzf#4007] [Q&A]
How can I get fzf to open a file and remain open so that other files can be opened without having to run fzf again, achieve something similar to what is described with a script and a loop while true; do but aesthetically not pleasant as it generates a annoying blink Someone to help me with some idea?

**Comment by LangLangBart** [comment]
Both the `README`[^1] and the `fzf` man page cover this case.

```sh
man fzf | less --pattern 'COMMAND EXECUTION'
```

If you're still not finding a satisfying solution, could you share your command and your `fzf` version?

[^1]: [GitHub - junegunn/fzf: :cherry_blossom: A command-line fuzzy finder](https://github.com/junegunn/fzf#executing-external-programs)
